### ISSUE TITLE
Install app through CLI - Timeout - Error: connect ETIMEDOUT

### ISSUE THREAD

**Issue by djanssen1** [athombv/homey#1572]
**What is your Homey version (Settings → System)?**

1.3 RC7

**What did you try to achieve?**

Install app through CLI.

2/3 - Uploading to Homey @ 192.168.97.117:80...
Error: connect ETIMEDOUT 192.168.97.117:80

Getting time out error every time. On Slack one other user has experienced this before but with him it got resolved after some time.

**Please attach a Homey System log (Settings -> System -> Send Log)**

967B9E8A62

**Comment by WeeJeWel** [comment]
Try running `athom homey --select` again. Your Homey's IP might have changed.

**Comment by djanssen1** [comment]
I didn't expected that it would be root cause as then it should give another warning or at least it would not take that long before it gives timeout. But I don't know the logic behind it. Anyway IP has not changed and is same. I tried it again after I came home at just before midnight from work and tried it again and it worked... so seems to be some kind of bug for the watch list. One other user has reported same issue and with him it was resolved also somehow after a while. Did a send log with code FB8653392D
